css/style.css

```diff
@@ -0,0 +1,25 @@
+body {
+    color: white;
+    background-color: #65659e;
+}
+/* 
+#tittle {
```

Comment:
Avoid commented code, also add 2 spaces as indentation

Reply:
It would be better if you try to phrase the comments politely, as suggestions instead of orders.
For example:

This is a polite suggestion: "I think it would be better to use semantic tags"

This reads as an order: "Use semantic tags"